docs/modules/ROOT/pages/how-tos/update_maintenance.adoc

```diff
@@ -24,10 +36,40 @@ Let the OpenShift operators do their job.
 ----
 kubectl --as cluster-admin get clusterversion version --watch
 ----
++
+. Check the final state via the `oc` commmand:
+[source,console]
+----
+$ oc adm upgrade --as cluster-admin
+Cluster version is 4.5.24
+
+No updates available. You may force an upgrade to a specific release image, but doing so may not be supported and result in downtime or data loss.
+----
 
 So far, that process just worked.
 In consequence we do lack the experience on how to observe and debug.
 Instruction will be added here, as soon as we gained that knowledge.
 As a starting point on the process, checkout https://docs.openshift.com/container-platform/latest/updating/updating-cluster-between-minor.html[Updating a cluster between minor versions] of the OpenShift 4 documentation.
 
 Also have a look at the blog post https://www.openshift.com/blog/the-ultimate-guide-to-openshift-release-and-upgrade-process-for-cluster-administrators[The Ultimate Guide to OpenShift Release and Upgrade Process for Cluster Administrators] which is an excellent source to understand the process.
+
+= Troubleshooting
+
+Get some more detailed information about the current state of the upgrade:
+
+[source,console]
+----
+kubectl --as cluster-admin get clusterversions.config.openshift.io version -o jsonpath={.status.conditions}
+----
+
+[NOTE]
+--
+In general: In the case of a error or warning, try to get more detailed information from the log from the specific operator or controller. Be patient as some warnings are just temporary and the operators might be able to release themself from a degraded state!
```

Comment:
```suggestion
In general: In the case of an error or warning, try to get more detailed information from the log from the specific operator or controller. Be patient as some warnings are just temporary and the operators might be able to relieve themselves from a degraded state!
```

It would be helpful to point the reader to the `specific operator or controller`.